src/App.jsx

```diff
@@ -14,57 +15,66 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
```

Comment:
The initial state of `selectedGood` is set to 'Jam'. Consider whether this is the desired default or if it should be an empty string to indicate no selection initially.